net.sourceforge.GTKWave.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Tony Bybell <bybell@rocketmail.com> -->
+<component type="desktop">
+  <id>net.sourceforge.gtkwave</id>
```

Comment:
this is case sensitive, therefore incorrect.

Please use the appdata file from upstream and patch as necessary. (and if you are upstream make sure the patch is in the next release)